rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -454,3 +454,7 @@ maas_venv_bin: "{{ maas_venv }}/bin"
 # Set this to enable or disable installing in a venv
 maas_venv_enabled: true
 
+#
+# Default horizon site name
+#
+horizon_site_name: "openstack dashboard"
```

Comment:
@sigmavirus24 I agree but It is supposed to be overridden if horizon_site_name is set in user variables though so I think it should probably stay the same.